adapters/oidc/adapter-core/pom.xml

```diff
@@ -95,6 +95,11 @@
             <artifactId>httpclient</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-validator</groupId>
```

Comment:
Please don't introduce new dependencies for a simple case like this.

Reply:
Removed the dependencies.

Reply:
Please check.